File: challenges/codeeval/easy/116/Skhorn.rb
#!/usr/bin/ruby
class MorseCode
   #@@no_of_customers=0
   def initialize()
      x = Array.new
      y = Array.new
      morse_dict = {'.-' => 'A',
       '-...' => 'B', 
       '-.-.' => 'C',
       '-..' => 'D',
       '.' => 'E',
       '..-.' => 'F',
       '--.' => 'G',
       '....' => 'H',
       '..' => 'I',
       '.---' => 'J',
       '-.-' => 'K',
       '.-..' => 'L',
       '--' => 'M',
       '-.' => 'N',
       '---' => 'O',
       '.--.' => 'P',
       '--.-' => 'Q',
       '.-.' => 'R',
       '...' => 'S',
       '-' => 'T',
       '..-' => 'U',
       '...-' => 'V',
       '.--' => 'W',
       '-..-' => 'X',
       '-.--' => 'Y',
       '--..' => 'Z',
       '-----' => '0',
       '.----' => '1',
       '..---' => '2',
       '...--' => '3',
       '....-' => '4',
       '.....' => '5',
       '-....' => '6',
       '--...' => '7',
       '---..' => '8',
       '----.' => '9',
       '@' => ' ' }

      File.open(ARGV[0]).each_line do |line|
         line = line.gsub(/  /, " @ ")
         l1 = line.split(' ').map { |x| x }
         l1.each { |x| print morse_dict[x]}
         puts
         

      end 
   end
end

x = MorseCode.new()

File: challenges/codeeval/easy/113/Skhorn.py
#!/usr/bin/python

import sys

class multiply_list:

	words_array = []
	mult1, mult2 = [], [];
	output = ""
	with open(sys.argv[1], 'r') as test_cases:
		for line in test_cases:
			words_array = line.strip().split('|')
			mult1 = words_array[0].strip().split(' ')
			mult2 = words_array[1].strip().split(' ')
			#print mult1
			#print mult2

			for i in range(len(mult1)):

				output += str(int(mult1[i])*int(mult2[i])) + " "

			print output[:-1]

			output = ""
			#for word in words_array:
			


x = multiply_list()